runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -7718,7 +7720,7 @@ static bool simpleReadMonitor(TR::Node *node, TR::CodeGenerator *cg, TR::Node *o
    return true;
    }
 
-void J9::Power::TreeEvaluator::generateCheckForValueTypeMonitorEnterOrExit(TR::Node *node, TR::LabelSymbol *helperCallLabel, TR::Register *objReg, TR::Register *objectClassReg, TR::Register *tempReg, TR::Register *condReg, TR::CodeGenerator *cg)
+void J9::Power::TreeEvaluator::generateCheckForValueMonitorEnterOrExit(TR::Node *node, TR::LabelSymbol *helperCallLabel, TR::Register *objReg, TR::Register *objectClassReg, TR::Register *temp1Reg, TR::Register *temp2Reg, TR::Register *condReg, TR::CodeGenerator *cg, int32_t classFlag)
```

Comment:
Was there a reason for adding the `int32_t classFlag` parameter? It looks like the constant `J9_CLASS_DISALLOWS_LOCKING_FLAGS` is the only value that is ever passed in.

Reply:
The purpose of adding a `classFlag` parameter is to make `generateCheckForValueTypeMonitorEnterOrExit()` more generic for future flag changes, although at the moment only  `J9_CLASS_DISALLOWS_LOCKING_FLAGS` is used.

Reply:
Okay, that makes sense.